src/Teams/Models/MultipleQuestionAddModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Teams.Domain;
+
+namespace Teams.Models
+{
+    public class MultipleQuestionAddModel  
+    {
+        public string QuestionText { get; set; }
+        public ICollection<QuestionAnswer> QuestionAnswers { get; set; }
```

Comment:
I think it's not very important to know which answer first or second in array. We just need few answers. So I believe it can be changed to IEnumerable